hapistrano.cabal

```diff
@@ -51,6 +51,7 @@ executable hap
   build-depends:       base
                      , hapistrano
                      , base-compat
+                     , optparse-applicative >= 0.13 && < 0.14
```

Comment:
I'm going to tested with `optparse-applicative 0.12`.

However, I think the `Opt.defaultPrefs { prefShowHelpOnEmpty = True }`  isn't available in `optparse-applicative 0.11`. That is why I didn't use that version.

Take a look: [optparse-applicative](http://hackage.haskell.org/package/optparse-applicative-0.11.0.2/docs/Options-Applicative-Extra.html#t:ParserPrefs)